AfricasTalkingTalent/TechnicalChallengeJune2019
# Task 1

**What does this pr do?**

Adds functionality where one can iterate over a list files and check whether the lines in the files contains a word with letter "**a**"

**How to manually test** 
_Please note that this project requires [python3](https://www.python.org/downloads/) to work properly_
- clone the repo
- make sure you have [python 3](https://www.python.org/downloads/) installed
- run `python code.py`

**How to test**
- run `python test_code.py`
